plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConnectorTest.java

```diff
@@ -238,6 +238,14 @@ public void testCreateTableAsSelect()
                 .hasStackTraceContaining("This connector does not support creating tables with data");
     }
 
+    @Test
+    @Override
+    public void testCreateTableAsSelectSchemaNotFound()
+    {
+        assertThatThrownBy(super::testCreateTableAsSelectSchemaNotFound)
+                .hasStackTraceContaining("This connector does not support creating tables with data");
```

Comment:
remove the test override, this should be handed by the check of the conenctor behavior